src/main/fc/init.c

```diff
@@ -208,40 +208,6 @@ static IO_t busSwitchResetPin        = IO_NONE;
 }
 #endif
 
-bool requiresSpiLeadingEdge(SPIDevice device)
```

Comment:
Where has the leding edge configuration gone?

Reply:
The rx_spi and sd_card drivers set `dev.busType_u.spi.leadingEdge = true` and the bus driver will then set their clock phase/polarity correctly as required. A bit messy, so I've added a `spiSetClkPhasePolarity()` routine to the API to control this.